main.go

```diff
@@ -804,7 +804,7 @@ func main() {
 		fmt.Printf(serviceInfo())
 		addr := fmt.Sprintf(":%d", *port)
 		http.HandleFunc("/", welcome)
-		http.ListenAndServe(addr, http.HandlerFunc(router))
+		log.Fatal(http.ListenAndServe(addr, http.HandlerFunc(router)))
```

Comment:
This will work. However, you should be using the `logError` logger that is initialized so that the error goes to the right place as configured by the corresponding flag.